apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/mechanism_apic.py

```diff
@@ -358,6 +360,13 @@ def is_port_promiscuous(port):
                 details['endpoint_group_name'])
         details.update(
             self.get_vrf_details(context, vrf_id=network['tenant_id']))
+        try:
+            self.apic_manager.vmm_shared_secred
```

Comment:
I guess you'll get KeyError in attestator anyways if the key is not present.

Also, there is a typo in the name: should be '...secret' instead of '...secred'